unlock-app/src/components/interface/buttons/lock/Save.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
```

Comment:
Not sure shy you need this new component? Couldn't you plug that logic in the `Buttons.Save` button?